### PR TITLE
Introduce CodeQL security scans

### PR DESCRIPTION
**Pull request by gwenneg**
Similar to https://github.com/RedHatInsights/notifications-backend/pull/149.

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    if: github.repository == 'RedHatInsights/notifications-frontend'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```
Comment (josejulio) [inline .github/workflows/codeql-analysis.yml:30]: Is this actually analyzing typescript? Should we see "hits" on this PR?

Comment (gwenneg) [reply .github/workflows/codeql-analysis.yml:30]: Yes it analyzes typescript when the matrix contains `javascript`.

**Comment by josejulio** [comment]
Mmh... I don't know why, but `ŧravis` didn't run anything.

**Comment by gwenneg** [comment]
Maybe changes to files under `.github/workflows` are ignored by Travis?

**Pull request merged**